bibcheck/plugins/journal_names_var.py

```diff
@@ -0,0 +1,161 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""
+Checking records in journals collection
+exactly one full name (130__a)
+exactly one short name (711__a)
+normalized short name is in name variants (730__a)
+all codens (030__a) and name variants are unique
+Option sort_variants will sort 730 fields by length.
+"""
+def normalize_name(name):
+    """No dots, all-caps"""
+    return name.replace('. ', ' ').replace('.', ' ').strip().upper()
+
+def sorted_keys(dd):
```

Comment:
[k for k, _ in sorted(dd.items(), key=itemgetter(1))]

or use lambda in the key function if you don't want to `from operator import itemgetter`

this is more stable, works in python3, doesn't require the values to be of equal lenght

Reply:
Thanks!
This is a line I don't understand - way beyond my Fortran-based programming skills.
But it seams to do what want.